burn-common/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+authors = ["Dilshod Tadjibaev (@antimora)"]
+edition = "2021"
+license = "MIT/Apache-2.0"
+name = "burn-common"
+readme = "README.md"
+repository = "https://github.com/burn-rs/burn-common"
+version = "0.1.0"
```

Comment:
For now I set the same version for all crates in the workspace, it's easier when deploying, so I would change it to 0.5.0. Maybe we could rethink a versionning strategy, but in another PR with the proper changes in the CI/CD pipeline.

Reply:
We can keep the version number in Cargo's workspace definition just like this: https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table